l10n_fr_department_delivery/model/delivery.py

```diff
@@ -0,0 +1,66 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    l10n FR Departments Delivery module for Odoo
+#    Copyright (C) 2015 Akretion (http://www.akretion.com)
+#    @author Alexis de Lattre <alexis.delattre@akretion.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class DeliveryGrid(models.Model):
+    _inherit = 'delivery.grid'
+
+    department_ids = fields.Many2many(
+        'res.country.department', string="Departments")
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = 'delivery.carrier'
+
+    @api.multi
+    def grid_get(self, contact_id):
+        '''This is almost a copy-paste of the native grid_get() method
+        from odoo/addons/delivery/delivery.py
+        I didn't find a way to inherit this method via super() without too
```

Comment:
At least can you please mark the beginning/end of the part you changed,  so we can upgrade the copy once in a while?